templates/frameworks/list_with_vue.html.twig

```diff
@@ -0,0 +1,504 @@
+                            <li><p class="govuk-!-margin-bottom-1">clicking 'start now' and answering a few simple questions to find the best agreement for your needs</p></li>
+                        </ul>
+                        <br>
+                        <p>If you are still unable to find what you need, contact our customer service centre using our <a class="govuk-link" href="/contact">contact us form</a>.</p>
+                    {% endif %}
+
+
+                    {% set statusData = statuses is defined and statuses is not empty ? {'statuses' : statuses} : {} %}
+                    {% set queryData = query is defined and query|trim is not empty ? {'q' : query} : {} %}
+
+                    {# Pagination #}
+                    {% if pillar is defined and pillar|trim is not empty %}
+                        {%  set pagination_data = {'pagination': pagination, 'route': 'frameworks_search', 'options': {'pillar': pillar_slug}|merge(statusData)|merge(queryData)} %}
+                    {% elseif category is defined and category|trim is not empty %}
+                        {%  set pagination_data = {'pagination': pagination, 'route': 'frameworks_search', 'options': {'category': category_slug}|merge(statusData)|merge(queryData)} %}
+                    {% elseif query is defined and query|trim is not empty %}
+                        {%  set pagination_data = {'pagination': pagination, 'route': 'frameworks_search', 'options': {'q': query}|merge(statusData)} %}
+                    {% else %}
+                        {%  set pagination_data = {'pagination': pagination, 'route': 'frameworks_search', 'options': statusData} %}
+                    {% endif %}
+
+                    {% if pagination.totalPages > 1 %}
+                        {{ include('includes/pagination.html.twig', pagination_data) }}
+                    {% endif %}
+
+                </div>
+            </div>
+
+
+        </main>
+    </div>
+    <script> searchFilterAgreement(); </script>
+
+    {% if app.environment == "prod" %}
+        <script src="/assets/scripts/libraries/vue.min.js?v=1"></script>
+    {% else %}
+        <script src="/assets/scripts/libraries/vue.js?v=1"></script>
+    {% endif %}
+    <script>
+
+        function supportsLiterals() {
+            try{
+                return eval("''===``");
+            }
+            catch(e){
+                return false;
+            }
+        }
+
+        if(!('Promise' in window) || !supportsLiterals() || !window.fetch) {
+            throw new Error("Your browser doesn't support the necessary JavaScript functions to use the enhanced search and has been served a non-JavaScript version");
+        }
+
+
+        var apiBaseUrl                  = '{{ api_base_url }}';
+        var appBaseUrl                  = '{{ app_base_url }}';
+        var pageNumber                  = {% if pageNumber is defined and pageNumber is not empty %}{{ pageNumber }}{% else %}1{% endif %};
```

Comment:
Yeah, this works. You just need to be careful with how to pass in the array. And I think I will change things up a bit instead of using string as an array

`var checkedStatus   = JSON.parse('{{ filters.checkedStatus|json_encode|e("js") }}');`